src/model/channel/message.rs

```diff
@@ -264,6 +264,32 @@ impl Message {
         cache_http.http().as_ref().delete_message_reactions(self.channel_id.0, self.id.0).await
     }
 
+    /// Remove reaction from the message given an [`Emoji`] or unicode character
```

Comment:
```suggestion
    /// Deletes the given [`Reaction`] from the message.
```
Adjusted wording to mirror docs for `ChannelId::delete_reaction`.